addons/account/models/chart_template.py

```diff
@@ -538,6 +538,11 @@ class AccountTaxTemplate(models.Model):
         string='Tax Received Account',
         domain=[('deprecated', '=', False)],
         help='Account used as counterpart for the journal entry, for taxes eligible based on payments.')
+    cash_basis_base_account_id = fields.Many2one(
+        'account.account',
```

Comment:
you mean account.account.template, right? o_O

Reply:
ho... yeah! Shame on me!

Reply:
I will use a smoke grenade to make that disappear